models/link.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+module.exports = function(sequelize, DataTypes) {
+  var link = sequelize.define('link', {
+    url: DataTypes.TEXT,
+    validate: {
```

Comment:
I had to remove the validate from the model to get it to work, the front end html validation on input type works fine to make sure it's in the http:// format